source/Tool/ConfigImpl.cpp

```diff
@@ -78,11 +78,11 @@ ConfigImpl(
     namespace path = llvm::sys::path;
 
     if(! files::isAbsolute(workingDir_))
-        throw Error("path \"{}\" is not absolute", workingDir_);
+        throw Error("working path \"{}\" is not absolute", workingDir_);
     workingDir = files::makeDirsy(files::normalizePath(workingDir_));
 
     if(auto err = files::requireDirectory(addonsDir_))
-        throw Error("path \"{}\" is not absolute", addonsDir_);
+        throw Error("addons path \"{}\" is not absolute", addonsDir_);
```

Comment:
I don't understand...how does this fix a runtime errr?

Reply:
it helps me figure out which path is failing.